Tests/ComposableArchitectureTests/EffectCancellationTests.swift

```diff
@@ -197,10 +197,11 @@ final class EffectCancellationTests: XCTestCase {
       DispatchQueue.global(qos: .userInteractive),
       DispatchQueue.global(qos: .utility),
     ]
+    let ids = (1...10).map { _ in UUID() }
```

Comment:
not related to this PR, but trying to fix more test leakage. again this doesn't affect users of the library, just the library tests themselves. :/